tests/test_pyiges.py

```diff
@@ -14,32 +15,526 @@ def impeller():
     return pyiges.read(examples.impeller)
 
 
+@pytest.fixture(scope='module')
+def surf(impeller):
+    return impeller.bspline_surfaces()[0] # pyiges.geometry.RationalBSplineSurface
+
+
+@pytest.fixture(scope='module')
+def curve(impeller):
+    return impeller.bsplines(as_vtk=False)[0] # geometry.RationalBSplineCurve
+
+
+@pytest.fixture(scope='module')
+def point(sample):
+    return sample.points(as_vtk=False, merge=False)[0] # pyiges.geometry.Point
+
+
+@pytest.fixture(scope='module')
+def carc(impeller):
+    return impeller.circular_arcs()[0] # pyiges.geometry.CircularArc
+
+
+@pytest.fixture(scope='module')
+def trafo(impeller):
+    return impeller._entities[172] # pyiges.geometry.Transformation
+
+
+@pytest.fixture(scope='module')
+def line(impeller):
+    return impeller.lines()[0] # pyiges.geometry.Line
+
+
+@pytest.fixture(scope='module')
+def entity(impeller):
+    return impeller._entities[0] # pyiges.entity.Entity
+
+
 def test_str(sample):
     assert 'Number of Entities: 5' in str(sample)
 
 
 def test_get_item(sample):
     assert '0.0, 0.0, 0.0' in str(sample.items[0])
+    assert '0.0, 0.0, 0.0' in str(sample[1])
+
+
+def test_parse_completeness1(impeller):
+    assert len(impeller.bspline_surfaces()) == 247
+    assert len(impeller.bsplines()) == 2342
+    assert len(impeller.circular_arcs()) == 468
+    assert len(impeller.conic_arcs()) == 0
+    assert len(impeller.edge_lists()) == 0
+    assert len(impeller.faces()) == 0
+    assert len(impeller.lines()) == 98
+    assert len(impeller.loops()) == 0
+    assert len(impeller.points()) == 0
+    assert len(impeller.vertex_lists()) == 0
+
+
+def test_parse_completeness2(sample):
+    assert len(sample.bspline_surfaces()) == 0
+    assert len(sample.bsplines()) == 0
+    assert len(sample.circular_arcs()) == 0
+    assert len(sample.conic_arcs()) == 0
+    assert len(sample.edge_lists()) == 0
+    assert len(sample.faces()) == 0
+    assert len(sample.lines()) == 0
+    assert len(sample.loops()) == 0
+    assert len(sample.points()) == 4
+    assert len(sample.vertex_lists()) == 0
+
+
+@pytest.mark.parametrize('merge', [False, True])
+def test_iges_to_vtk1(impeller, merge):
+    items = impeller.to_vtk(delta=0.5, merge=merge)
+    assert items.bounds == pytest.approx((-50.516562808,  49.445007453, -49.510005208,
```

Comment:
I used `pytest.approx` assuming that a recent `pytest` version can be required, let me know if that's a problem.

Reply:
Good choice. `pytest.approx` has been around for long enough